pkgs/by-name/ai/aider-chat/package.nix

```
@@ -0,0 +1,114 @@
+{
+  lib,
+  stdenv,
+  python311,
+  fetchFromGitHub,
+  gitMinimal,
+  portaudio,
+}:
+
+let
+  python3 = python311.override {
+    self = python3;
+    packageOverrides = _: super: { tree-sitter = super.tree-sitter0_21; };
```

```suggestion
    packageOverrides = _: super: { tree-sitter = super.tree-sitter_0_21; };
```
and please rebase on master.

---

Oops it slipped in again. Sorry about that.